### ISSUE TITLE
Bad keyword error

### ISSUE THREAD

**Issue by Burhan-Q** [instructor-ai/instructor#1253]
- [x] This is actually a bug report.
- [ ] I am not getting good LLM Results
- [ ] I have tried asking for help in the community on discord or discussions and have not received a response.
- [x] I have tried searching the documentation and have not found an answer.

**What Model are you using?**

- [ ] gpt-3.5-turbo
- [ ] gpt-4-turbo
- [ ] gpt-4
- [x] Other (Claude, but error occurs before query)

**Describe the bug**
Apologies, I haven't had a ton of time to investigate, but I wanted to open a ticket before I forgot. This might _actually_ be an error from a third party package. I was following this guide https://python.useinstructor.com/integrations/anthropic/ and when running 

```py
client = instructor.from_anthropic(
    anthropic.Anthropic(),
)
```
an error is raised for a bad keyword `proxies` for `Client.__init__()` and is unable to create the `client` object.

**To Reproduce**
Follow the guide

**Expected behavior**
Client created without error.

**Patch**
I _was_ able to sidestep the issue by commenting L860 in `_base_client.py` but it's not obvious to me if that would cause downstream issues. After commenting this line, I was able to create the `client` object successfully and was able to send a query, however there were other issues but I have not ruled out external factors (my structured outputs) as the cause for those, as I ran out of time.

- Ubuntu 22.02
- Python 3.11
- Installed using `uv pip install "instructor[anthropic]"` command in dedicated project `venv`

Sorry for the rushed post. Hackathon followed by cross country travel.

**Comment by a3lem** [comment]
Appears to be an issue with the Anthropic SDK, since simply creating an instance of the Anthropic() client triggers the same error.

It's been fixed in the latest version of the Anthropic SDK (v0.40). https://github.com/anthropics/anthropic-sdk-python/issues/779

Instructor should support that version but UV is complaining about a conflict. 

The error arises from httpx. Fastest workaround is to downgrade, as suggested in a comment below the anthropic issue.

**Comment by Burhan-Q** [comment]
Happy to close this issue as resolved, but thought it could also be helpful for other users looking for answers on the same topic. Any preference?

**Comment by ivanleomk** [comment]
Have a tentative PR here that will bump anthropic up to 0.42.0 and make it the default version for people moving forward. This also allows us to support prompt caching in the normal standard message.create.

Will start working on documentation for computer use in the beta #1281

**Comment by ivanleomk** [comment]
Ok I've just published instructor 1.7.2 which bumps anthropic to v0.42.0 and avoids this issue with https
